gatekeeper/bootstrap/aws/aws.go

```diff
@@ -0,0 +1,204 @@
+// Package aws provides Gatekeeper authentication for AWS
+package aws
```

Comment:
This file contains logic for both  the client side and the server side. Its not immediately clear to me when reading this where each responsibility lies. Maybe that separation could be clarified?